src/components/AgenciesPage/components/AgenciesList.tsx

```diff
@@ -1,14 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Grid } from '@mui/material';
 import { AgenciesItem } from './components';
-import { agenciesData } from '../mocks/agenciesData';
-
-export const AgenciesList = () => (
-  <Grid container>
-    {agenciesData.map(agency => (
-      <Grid item key={agency.id} xs={12} sm={6} md={4} lg={3}>
-        <AgenciesItem agency={agency} />
-      </Grid>
-    ))}
-  </Grid>
-);
+import { fetchAgencies } from '../services/fetchAgencies';
+import { Agency } from '../types/Agencies';
+
+export const AgenciesList = () => {
+  const [data, setData] = useState<Agency[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const response = await fetchAgencies();
+      setData(response);
+    };
+
+    fetchData();
+  }, []);
+
+  if (!data.length) return null;
+
+  return (
+    <Grid container>
+      {data.map(agency => (
```

Comment:
Better to use more readable names for arrays, in your case data will change on agencies

Reply:
I